api/src/pcapi/core/educational/api/offer.py

```diff
@@ -256,7 +256,7 @@ def create_collective_offer(
         subcategoryId=offer_data.subcategory_id,
         students=offer_data.students,
         contactEmail=offer_data.contact_email,
-        contactPhone=offer_data.contact_phone,
+        contactPhone=offer_data.contact_phone or "",
```

Comment:
or None plutot non ?

Reply:
mypy est pas très content quand je mets None : 
`src/pcapi/core/educational/api/offer.py:259:22: error: Argument "contactPhone" to "CollectiveOffer"
has incompatible type "Optional[str]"; expected "str"  [arg-type]
            contactPhone=offer_data.contact_phone or None,`
            
Je rate peut être qq chose ?

Reply:
C'est pas à cause de `contactPhone: str = sa.Column(sa.Text, nullable=True)` ?

Reply:
Yes mais on en a besoin pour créer une offre collective sans contactPhone

Reply:
Désolé, c'était pas clair. C'est pas parce que tu forces le type à `str` ? Il faudrait pas `str | None` ?

Reply:
Yes mieux en effet